extensions/examples/L1T-catalog.json

```diff
@@ -0,0 +1,113 @@
+{
+  "name": "Landsat catalog",
+  "description": "This is a very basic sample catalog.",
+  "links": [
+    {
+      "href": "http://landsat-pds.s3.amazonaws.com/L8/landsat8-sample.json",
+      "rel": "item"
+    },
+    {
+      "href": "http://landsat-pds.s3.amazonaws.com/L8/L1T-catalog.json",
+      "rel": "self"
+    }
+  ],
+  "properties": {
```

Comment:
I wouldn't change to the new catalog.json inherited metadata model in this PR as this is not the topic of this PR. That should be separated.